Bio/PDB/PDBParser.py

```diff
@@ -86,7 +86,10 @@ def get_structure(self, id, file):
             self.structure_builder.init_structure(id)
 
             with as_handle(file, mode="rU") as handle:
-                self._parse(handle.readlines())
+                lines = handle.readlines()
```

Comment:
I didn't realise the PDB parser loaded everything into memory like this - rather than a line-by-line approach.

Reply:
The PDB parser is problematic in many ways, and personally I think it should be overhauled completely (as João et al. also suggested recently).

Reply:
Yes, there is a lot that could be done differently here with a fresh start - something they talked about in Basel at the BOSC CoFest 2019.